components/tools/OmeroPy/src/omero/install/jvmcfg.py

```diff
@@ -398,6 +398,12 @@ def adjust_settings(config, template_xml,
             ("pixeldata", pixeldata), ("repository", repository))
 
     for name, StrategyType in loop:
+        if name not in options:
+            raise Exception(
+                "Cannot find %s option. Make sure templates.xml was "
+                "not copied from a 5.0.2 or older server" % name)
```

Comment:
If so, maybe mention in upgrade notes a link to the proper way to change the settings.